lib/workers/repository/init/inherited.spec.ts

```diff
@@ -84,4 +85,86 @@ describe('workers/repository/init/inherited', () => {
     expect(res.onboarding).toBeFalse();
     expect(logger.warn).not.toHaveBeenCalled();
   });
+
+  it('should resolve presets found in inherited config', async () => {
+    platform.getRawFile.mockResolvedValue(
+      '{"onboarding":false,"labels":["test"],"extends":[":automergeAll"]}',
+    );
+    const res = await mergeInheritedConfig(config);
+    expect(res.labels).toEqual(['test']);
+    expect(res.onboarding).toBeFalse();
+    expect(logger.warn).not.toHaveBeenCalled();
+    expect(logger.debug).toHaveBeenCalledWith(
+      'Resolving presets found in inherited config',
+    );
+  });
+
+  it('should warn if presets fails validation with warnings', async () => {
+    platform.getRawFile.mockResolvedValue(
+      '{"onboarding":false,"labels":["test"],"extends":[":automergeAll"]}',
+    );
+    jest
+      .spyOn(validation, 'validateConfig')
+      .mockResolvedValueOnce({
+        warnings: [],
+        errors: [],
+      })
+      .mockResolvedValueOnce({
+        warnings: [
+          {
+            message: 'some warning',
+            topic: 'Configuration Error',
+          },
+        ],
+        errors: [],
+      });
+    const res = await mergeInheritedConfig(config);
+    expect(res.binarySource).toBeUndefined();
+    expect(logger.warn).toHaveBeenCalledWith(
+      {
+        warnings: [
+          {
+            message: 'some warning',
+            topic: 'Configuration Error',
+          },
+        ],
+      },
+      'Found warnings in presets inside the inherited configuration.',
+    );
+  });
+
+  it('should throw error if presets fails validation with errors', async () => {
+    platform.getRawFile.mockResolvedValue(
+      '{"labels":["test"],"extends":[":automergeAll"]}',
+    );
+    jest
+      .spyOn(validation, 'validateConfig')
+      .mockResolvedValueOnce({
+        warnings: [],
+        errors: [],
+      })
+      .mockResolvedValueOnce({
+        warnings: [],
+        errors: [
+          {
+            message: 'some error',
+            topic: 'Configuration Error',
+          },
+        ],
+      });
+    await expect(mergeInheritedConfig(config)).rejects.toThrow(
+      CONFIG_VALIDATION,
+    );
+    expect(logger.warn).toHaveBeenCalledWith(
+      {
+        errors: [
+          {
+            message: 'some error',
+            topic: 'Configuration Error',
+          },
+        ],
+      },
+      'Found errors in presets inside the inherited configuration.',
```

Comment:
`Found errors` only tells me something went wrong, but not _what_ went wrong.

Should we print the error message in the log message?

Reply:
We do: https://github.com/RahulGautamSingh/renovate/blob/140a794f73d908c134c17afcf1845f96fae57440/lib/workers/repository/init/inherited.ts#L121

Reply:
Thank you for checking! :wink:

I'll let a maintainer resolve my review comment, I can't do that myself...